packages/datetime/src/datePickerNavbar.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2015 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import { Button } from "@blueprintjs/core";
+import * as React from "react";
+import { NavbarElementProps } from "react-day-picker/types/props";
+
+import classNames from "classnames";
+import * as Classes from "./common/classes";
+import { areSameMonth } from "./common/dateUtils";
+
+export interface IDatePickerNavbarProps extends NavbarElementProps {
+    maxDate: Date;
+    minDate: Date;
+}
+
+export class DatePickerNavbar extends React.PureComponent<IDatePickerNavbarProps> {
+    public render() {
+        const { classNames: classes, month, maxDate, minDate } = this.props;
+
+        return (
+            <div className={classNames(Classes.DATEPICKER_NAVBAR, classes.navBar)}>
```

Comment:
Where does `classes.navBar` come from / what is its value? Not seeing it in the changed lines of the parent component(s). I assume it's just the `ReactDayPicker` default, and we need to provide it in order for basic ReactDayPicker styles to be applied in addition to our style overrides?

Reply:
DayPicker itself. it's the actual `.DayPicker-whatever` class.